apis/stardust/types/index.d.ts

```diff
@@ -411,6 +412,16 @@ declare namespace stardust {
          */
         convertTo(newType: string, forceUpdate?: boolean, forcePatch?: boolean): Promise<object>;
 
+        /**
+         * Toggles the chart to a data view of the chart.
+         * 
+         * The chart will be toggled to the type defined in the nebula context (dataViewType).
+         * 
+         * The default dataViewType for nebula is sn-table. The specified chart type needs to be registered as well in to make it possible to render the data view.

```

Comment:
I assume you mean this?
```suggestion
         * The default dataViewType for nebula is sn-table. The specified chart type needs to be registered as well, in order to make it possible to render the data view.

```